e2e/test/scenarios/dashboard-filters/reproductions/8030-32444-reload-card-without-change.cy.spec.js

```diff
@@ -57,6 +76,38 @@ describe("issue 8030", () => {
   });
 });
 
+describe("issue 32444", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should not reload dashboard cards with filter not connected to a filter (metabase#32444)", () => {
```

Comment:
> with filter not connected to a filter

Probably a typo? Should it be something like "with a dashcard not connected to a filter" or vice-versa?

Reply:
nice catch!